parsec/core/sync_monitor.py

```diff
@@ -105,19 +105,19 @@ def _on_entry_updated(event, workspace_id=None, id=None):
 
         for id, (first_updated, last_updated) in updated_entries.items():
             if now - first_updated > MAX_WAIT:
-                if id == self.fs.user_fs.user_manifest_access.id:
-                    await self.fs.sync("/")
+                if id == self.user_fs.user_fs.user_manifest_access.id:
+                    await self.user_fs.sync("/")
                 else:
-                    await self.fs.sync_by_id(id)
+                    await self.user_fs._syncer.sync_by_id(id)
```

Comment:
The `_syncer` object is legacy and shoudn't be used directly. `sync_by_id` is available as a method of `WorkspaceFS` but you would also need the workspace id to get the corresponding instance.